src/canvas/SelectableCanvas.ts

```diff
@@ -1503,6 +1503,7 @@ export class SelectableCanvas<
     wrapperEl.removeChild(lowerCanvasEl);
     this.contextCache = null;
     this.contextTop = null;
+    // TODO: interactive canvas should NOT be used in node, therefore there is no reason to cleanup node canvas
```

Comment:
well for some weird reason people could want to draw a picture of something with controls.
I wouldn't argue over people intentions/needs. If it works it works, if it can't work it can't work.

Reply:
A weird concept indeed
So you are for... OK
I thought we wanted to reduce the bundle but to say the truth I don't care.
One day we can think about it. So I should add add streaming to canvas as well.